### PR TITLE
unbound: migrate active/outgoing interfaces discarding invalid values

### PR DESCRIPTION
**Pull request by fichtner**
As discussed.

### CODE REVIEW

**Comment by AdSchellevis** [comment]
@fichtner that’s the trick 😊 nice!

File: src/opnsense/mvc/app/models/OPNsense/Unbound/Migrations/M1_0_8.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+namespace OPNsense\Unbound\Migrations;
+
+use OPNsense\Base\BaseModelMigration;
+use OPNsense\Core\Config;
```
Comment (fichtner) [inline src/opnsense/mvc/app/models/OPNsense/Unbound/Migrations/M1_0_8.php:32]:
```suggestion
```

File: src/opnsense/mvc/app/models/OPNsense/Unbound/Migrations/M1_0_5.php [hunk 2]
```diff
@@ -53,6 +53,9 @@ public function run($model)
         $new['enabled'] = isset($config->unbound->enable) ? '1' : '0';
 
         $model->general->setNodes($new);
+
+        $model->general->active_interface->normalizeValue();
```
Comment (fichtner) [inline src/opnsense/mvc/app/models/OPNsense/Unbound/Migrations/M1_0_5.php:57]:
```suggestion
        /* discard missing interfaces to pass validation */
        $model->general->active_interface->normalizeValue();
```

**Comment by swhite2** [comment]
looks good :)

**Pull request merged**